undetected_chromedriver/__init__.py

```diff
@@ -782,7 +782,11 @@ def quit(self):
                 else:
                     logger.debug("successfully removed %s" % self.user_data_dir)
                     break
-                time.sleep(0.1)
+                
+                try:
+                    time.sleep(0.1)
+                except OSError:
+                    pass
```

Comment:
To be a bit more specific about what this is handling:
```suggestion
                except OSError as ex:
                    if getattr(ex, "winerror", None) == 6:
                        pass
                    else:
                        raise ex
```